staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,57 @@
+*{
```

Comment:
MOVE common css rules (mobile / desktop) to main.css

Reply:
done!